examples/llm_compression/openvino/tiny_llama_find_hyperparams/main.py

```diff
@@ -63,6 +64,7 @@ def compress_model(
         group_size=group_size,
         awq=awq,
         sensitivity_metric=nncf.parameters.SensitivityMetric.MAX_ACTIVATION_VARIANCE,
+        advanced_parameters=AdvancedCompressionParameters(statistics_dir_path="statistics"),
```

Comment:
@kshpv, did you check that the example returns the same results?

Reply:
Yes, I checked. It looks they are the same. test_examples build 561